search_params.py

```diff
@@ -177,6 +174,31 @@ def load_static_data(config):
     }
 
 
+def retrain_best_model(log_path, merge_train_val=False):
+    """Retrain the best model with the best hyperparameters.
+    The model will train on the whole training data if `merge_train_val` is True.
```

Comment:
```suggestion
    A new model is trained on the combined training and validation data if `merge_train_val` is True.
    If a test set is provided, it will be evaluated by the obtained model.
```